plugins/pulp_python/plugins/models.py

```diff
@@ -1,120 +1,85 @@
-from gettext import gettext as _
 import hashlib
-import re
-import tarfile
 
 from mongoengine import StringField
 from pulp.server.db.model import FileContentUnit
+import twine  # noqa
 
 from pulp_python.common import constants
 
 
 DEFAULT_CHECKSUM_TYPE = 'sha512'
-
-
-# These are required to be in the PKG-INFO file.
-REQUIRED_ATTRS = ('name', 'version', 'summary', 'home_page', 'author', 'author_email', 'license',
-                  'description', 'platform')
+REQUIRED_ATTRS = ('name', 'version', 'author', 'author_email', 'description', 'summary')
 
 
 class Package(FileContentUnit):
     """
     This class represents a Python package.
```

Comment:
It would be great if you documented this class more extensively. I know why you are using the filename as the unit key, but only because you told me in person. I'd like this doc-block to explain what each field is, where it comes from, and why it is there. You've done a lot of research I don't want to have to do myself :smile: